docs/src/modules/components/AppNavDrawer.js

```diff
@@ -54,6 +55,15 @@ function ProductDrawerButton(props) {
     setAnchorEl(null);
   };
 
+  const handleEventDelegation = (e) => {
```

Comment:
```suggestion
  const handleEventDelegation = (event) => {
```

just for consistency with the rest of the codebase